frontend/editor.css

```diff
@@ -167,6 +167,12 @@ pluto-output code {
     font-size: 0.75rem;
 }
 
+pluto-output .markdown code {
```

Comment:
What happens if you remove the `.markdown` selector?

Reply:
it will influence the normal output, say cell:
```
x = 3
```
the `3` will be displayed as if it's in markdown.